tests/test_canister_mo/mops.toml

```
@@ -1,7 +1,3 @@
 [dependencies]
-cbor = "0.1.3"
-ic-certification = "0.1.1"
-sha2 = "0.0.2"
-candid = "1.0.2"
 base = "0.9.1"
-ic-websocket-cdk = "https://github.com/omnia-network/ic-websocket-cdk-mo#a229d4a6d95987b4b1d46e9c5e7fde6d896fa118"
\ No newline at end of file
+ic-websocket-cdk = "0.1.0"
```

```suggestion
ic-websocket-cdk = "0.1.1"
```
The last bug fixes are in **v0.1.1**.